pallets/pallet-tfgrid/src/lib.rs

```diff
@@ -560,8 +559,8 @@ decl_module! {
             Ok(())
         }
 
-        #[weight = 10 + T::DbWeight::get().writes(1)]
-        pub fn report_uptime(origin, uptime: u64) -> dispatch::DispatchResult {
+        #[weight = (10 + T::DbWeight::get().writes(1) + T::DbWeight::get().reads(3), DispatchClass::Operational)]
+        pub fn report_uptime(origin, uptime: u64, used_resources: types::Resources) -> dispatch::DispatchResultWithPostInfo {
```

Comment:
The used resources can technically be calculated from the capacity reports sent to the chain. While I'm not objected to adding this, lets remove it here and add it in a separate PR, as these are 2 distinct pieces of functionality